iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm

```diff
@@ -50,10 +58,10 @@ - (instancetype)initWithRawData:(Info const &)rawData ohLocalization:(id<IOpenin
           _email = ToNSString(value);
           _emailUrl = [NSURL URLWithString:[NSString stringWithFormat:@"mailto:%@", _email]];
           break;
-        case MetadataID::FMD_CONTACT_FACEBOOK: _facebook = ToNSString(value); break;
-        case MetadataID::FMD_CONTACT_INSTAGRAM: _instagram = ToNSString(value); break;
-        case MetadataID::FMD_CONTACT_TWITTER: _twitter = ToNSString(value); break;
-        case MetadataID::FMD_CONTACT_VK: _vk = ToNSString(value); break;
+        case MetadataID::FMD_CONTACT_FACEBOOK: _facebook = denormalizaSocialContact(value, @"https://facebook.com/"); break;
```

Comment:
It would be great to avoid hardcoding these URL parts several times everywhere in the code.